portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,79 @@
+body {
+  margin:0;
+}
+
+#fullpage {
+  min-width: 100vw;
+  min-height: 100vh;
+  min-width: 100vw;
+  min-height: 100vh;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+.section {
+  min-height: 100vh;
+  max-height: 100vh;
+  overflow: hidden;
+  min-width: 100%;
+  max-width: 100%;
+}
+
+#intro,
+.intro-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.intro-wrapper {
+  width: 70%;
+  height: 50vh;
+  text-align:center;
+}
+
+.intro-wrapper > * {
+  display: flex;
+  flex-direction: column;
+  flex: 0.5;
+  align-items: center;
```

Comment:
[alphabetize declarations](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order)

Reply:
Alphabetised in commit c4c4699